oeps/oep-0004-arch-oauth-scopes.rst

```diff
@@ -10,7 +10,7 @@ OEP-4: Application Authorization (Scopes)
    * - Title
      - Application Authorization (Scopes)
    * - Last Modified
-     - 2016-10-13
+     - 2020-03-23
```

Comment:
Please create a separate PR for updating the existing Scopes OEP.  That will allow a closer review of these changes, separate from rolling out a new Authentication OEP.  I believe there are more changes needed for this existing OEP.

Reply:
The new PR is here: https://github.com/edx/open-edx-proposals/pull/150